server/botany/settings.py

```diff
@@ -23,8 +23,13 @@
 BOTANY_NUM_ROUNDS = int(os.environ["BOTANY_NUM_ROUNDS"])
 BOTANY_OPCODE_LIMIT = int(os.environ["BOTANY_OPCODE_LIMIT"])
 
-# TODO read this from environment (and make optional)
-BOTANY_TOURNAMENT_CLOSE_AT = datetime(2018, 9, 18, 14, 30, tzinfo=bst)
+BOTANY_TOURNAMENT_START_AT = os.environ.get("BOTANY_TOURNAMENT_START_AT")
```

Comment:
`os.environ` only contains strings. You'll need to parse the string into a `datetime` object here